app/models/queue_item.rb

```diff
@@ -0,0 +1,16 @@
+class QueueItem < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :video
+
+  delegate :category, to: :video
+  delegate :title, to: :video, prefix: :video
+
+  def rating
+    review = Review.where(user_id: user.id, video_id: video.id).first
```

Comment:
You should not use `where(...).first` to search your single record. Check out the [note in this assignment](https://launchschool.com/lessons/a3a6fbb9/assignments/1566).